rust/src/command/mod.rs

```diff
@@ -142,7 +142,11 @@ pub trait UserCommandWithStatusT {
 
     fn sender(&self) -> PublicKey;
 
+    fn receiver(&self) -> PublicKey;
+
     fn nonce(&self) -> u32;
+
+    fn amount(&self) -> u64;
 }
 
 impl UserCommandWithStatusT for UserCommandWithStatus {
```

Comment:
UserCommandWithStatusT?

Reply:
Trait for `UserCommand` functions. It was added with v2 blocks.